### DISCUSSION TITLE
global.json Proposal Draft Document

### DISCUSSION THREAD

**Discussion by mariam-abdulla** [microsoft/testfx#4316] [Ideas]
### General
This is a draft proposal document for designing the global.json file for specifying the test runner with which we will be running the tests in the test projects.

### Description

We have been working on improving the user experience in **dotnet test** by using Microsoft Testing Platform when running tests.
If you are interested in learning about the new testing platform, please check out this [link](https://learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-platform-intro?tabs=dotnetcli#microsofttestingplatform-pillars).

We want to allow users to opt-in/opt-out this new experience through a **global.json** file where you specify the test runner with which you want to run your tests.

### Suggestion

Please find below a sample of the global.json file

```
{
"test-sdk" :
  {
    "name": "testingplatform"
  }
}
```

**Comment by Evangelink** [comment]
Closing in favor of https://github.com/dotnet/sdk/issues/45471
